devtools/JenkinsOnDocker/fetchplugins.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Overall suggestion to this file: I think using a shell script for such operation is not the best idea. This is due to different versions of tools on various operation systems (BSD vs GNU sed etc, as below comments show). I think it would be better to actually parse the html document using python/groovy or any language (those two are preferred, because we already use them in other places). Such solution would be: 
 - more readable than regexes
 - OS agnostic 
 - easier to change for example if the page layout changes

Reply:
Additionally (I'm not 100000% sure about this idea, so feel free to criticize it ;) ): 

Because (AFAIK) the plugin list doesn't change that often, I think it might be good to include the plugins.txt list too as a form of a backup in case fetching fails or even instead fetching the plugins. The latter would spare adding the script for this but would require updating the plugins.txt file manually. WDYT?

Reply:
I have an email out to INFRA team regarding getting the list of plugins on available on Jenkins. I had this script as a starting point.

For now I'll do the following:
1) I'll remove the script 
2) prepare some feasible version of plugins.txt  and check it in for now. 
3) Once I get a more up-to-date version, I'll update this list.